### PR TITLE
Map markers

### PR DESCRIPTION
**Pull request by josh-hdz**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/GoogleMap.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            -100.3052989,
+            -121.9756271,
+            -103.3704326
+        };
+
+        this.apiKey = System.getenv("GOOGLE_MAPS_API_KEY");
+        this.markers = new ArrayList<Marker>();
+
+        for(int i = 0; i < label.length; i++){
+            this.markers.add(new Marker(label[i], lat[i], lng[i]));
```
Comment (jmuzy-sps) [inline portfolio/src/main/java/com/google/sps/data/GoogleMap.java]: Optional (I'm actually not sure what the best option here is, so this is kind of just my thought process, and if you have anything to add sweet, otherwise you can take no action): This method of three corresponding arrays works fine because you have a smaller data set that's likely not to change. But what if you had 100s of markers, or if you want to add another one? You'd have to make sure you add the three values (label, lat, lng) to the same index spot, which could be error prone. Or what if you need to update value 57 out of 123? You'd have to count 57 into each array

1) I'm not 100% sure how Datastore works, but I'd like to think you could add these points to a table there, and then just pull the values from your data table.

2) While maybe more work, another option would be to directly add the values as a Marker:
`new Marker("Chihuahua, Chih",  28.6709132, -106.1346578)`
This way, it's clear what's matched up with what and you could sort these in alphabetical order to make finding markers easier if you had to debug for some reason.

TLDR I think these values should be grouped somehow, but I'm not sure of the best way. I think 1) might be best if possible, but I'd probably just go with 2) for something low effort

**Review by jmuzy-sps — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/data/GoogleMap.java (hunk 1 above)
Comment (josh-hdz) [reply portfolio/src/main/java/com/google/sps/data/GoogleMap.java]: I was also not a fan of that either, I spent like 10mins placing all the right values in the position. It never occurred to me that option 2 was better in almost every way.

I'll do solution 2 for a quick refactor, but I'll defiantly deploy solution 1 next week for a long term solution.

**Review by jmuzy-sps — APPROVED** [review]
Nice job!
